examples/workflow-glsp/src/workflow-startup.ts

```diff
@@ -15,14 +15,14 @@
  ********************************************************************************/
 
 import { GridManager, IDiagramStartup } from '@eclipse-glsp/client';
-import { MaybePromise } from '@eclipse-glsp/sprotty';
+import { MaybePromise, TYPES } from '@eclipse-glsp/sprotty';
 import { inject, injectable, optional } from 'inversify';
 
 @injectable()
 export class WorkflowStartup implements IDiagramStartup {
     rank = -1;
 
-    @inject(GridManager) @optional() protected gridManager?: GridManager;
+    @inject(TYPES.IGridManager) @optional() protected gridManager?: GridManager;
```

Comment:
```suggestion
    @inject(TYPES.IGridManager) @optional() protected gridManager?: IGridManager;
```

If we are already introducing interfaces for the manager classes, we should only rely on those interfaces when injecting that class. This is true for all injection points for all manager classes.